crates/rpc/api/src/lib.rs

```diff
@@ -152,3 +154,13 @@ pub trait StrataSequencerApi {
     #[method(name = "strata_getTxStatus")]
     async fn get_tx_status(&self, txid: HexBytes32) -> RpcResult<Option<L1TxStatus>>;
 }
+
+/// Debug Specific Api
+#[cfg(feature = "debug-utils")]
+#[cfg_attr(all(feature = "debug-utils", not(feature = "client")), rpc(server))]
+#[cfg_attr(all(feature = "debug-utils", feature = "client"), rpc(server, client))]
+pub trait StrataDebugApi {
+    /// for exiting the client based on context
+    #[method(name = "stratadebug_bail")]
```

Comment:
I think it's actually fine to expose this RPC trait without requiring the feature flag, so keep the feature flag structure simpler.  It's the impl that we only want to enable if it's set.  If the feature isn't set then we can return with an error instead of actually setting up the abort.

Not completely sure if that's the perfect approach but this is always going to be slightly hacky.

In Puru's PR there's a `debug_` namespace that gets set up.  This would probably make sense to be in there as well actually.  If you rebase we can add it to that.